static/templates/sheets/actor/creature/creature-notes.hbs

```diff
@@ -1,17 +1,50 @@
 <section class="tab {{tab.cssClass}}" data-group="primary" data-tab="{{tab.id}}">
-    {{#if (isGM)}}
-        <div class="editor-section">
-            <div class="header">{{localize "GM Notes"}}</div>
-            {{formInput fields.details.fields.gmnotes.fields.value value=source.system.details.gmnotes.value enriched=enriched.system.details.gmnotes.value toggled=true}}
+
+    <div class="header-row">
+        <div class="header-group">
+            <input type="text" name="system.details.age.value" value="{{system.details.age.value}}">
+            <label>{{localize "Age"}}</label>
+        </div>
+        <div class="header-group">
+            <input type="text" name="system.details.height.value" value="{{system.details.height.value}}">
+            <label>{{localize "CharacterHeight"}}</label>
+        </div>
+        <div class="header-group">
+            <input type="text" name="system.details.weight.value" value="{{system.details.weight.value}}">
+            <label>{{localize "Weight"}}</label>
         </div>
+        <div class="header-group">
+            <input type="text" name="system.details.haircolour.value" value="{{system.details.haircolour.value}}">
+            <label>{{localize "Hair Colour"}}</label>
+        </div>
+        <div class="header-group">
+            <input type="text" name="system.details.eyecolour.value" value="{{system.details.eyecolour.value}}">
+            <label>{{localize "Eye Colour"}}</label>
+        </div>
+    </div>
+    <div class="header-row">
+        <div class="header-group">
+            <input type="text" name="system.details.distinguishingmark.value"
+                value="{{system.details.distinguishingmark.value}}">
+            <label>{{localize "Distinguishing Mark"}}</label>
+        </div>
+    </div>
+
+    {{#if (isGM)}}
+    <div class="editor-section">
+        <div class="header">{{localize "GM Notes"}}</div>
+        {{formInput fields.details.fields.gmnotes.fields.value value=source.system.details.gmnotes.value
+        enriched=enriched.system.details.gmnotes.value toggled=true}}
+    </div>
```

Comment:
I think I'm still going to say that these fields on the creature sheet aren't necessary.

Reply:
It would be awesome to have it consistent, especially that creatures are also orcs, goblins, gors etc, for which those stuff make sense. I think even Vampires are actual creature objects.

Reply:
It doesn't really follow the original (though not always followed) paradigm of NPCs are specific characters, while creatures are generic. 

There's always been talk about there not being much distinction between NPCs and Creatures, and I think these fields are decent as a "feature" of NPCs.

These fields are fine to stay in the standard model, but I don't see any benefit to them being on the sheet as they will be rarely, if ever, used for creatures.

Reply:
well. at least having them in standard model would partially help. there is always way to extend ui. if you are fine with that, i will remove the UI part changes, but anyway its :(

Reply:
Yeah go ahead and remove it, but as a consolation, I am currently considering letting NPCs and Creatures use either NPC or Creature sheet, so if someone wanted a Creature to have the NPC fields, they can simply switch to the NPC sheet. 

Also considering deprecating creatures and just having two different sheets for NPCs: NPC and Creature, not sure yet.

Reply:
done